web_src/css/user.css

```diff
@@ -175,4 +154,13 @@
 
 #notification_div .tab.segment {
   overflow-x: auto;
+  padding: 0;
+}
+
+#notification_div .menu .active.item {
+  background: var(--color-box-body);
```

Comment:
Why the background should be set separately? 

I guess all Dropdown menu items should share the same styles across the whole Gitea?

Reply:
Dropdown? This is a tab above a table, quite unique I'd say.

Reply:
Thought I do guess we could go for pull-style tabs here as well, like on the Releases/Tags page.

Reply:
If you only want to modify the "tab menu" styles, I think it's  better to do `#notification_div .tabular.menu`

Otherwise, if there is a dropdown inside, its style is affected by `.menu .active.item`